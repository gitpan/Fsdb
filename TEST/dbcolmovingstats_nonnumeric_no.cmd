prog='dbcolmovingstats'
args='-w 4 -m count'
cmp='diff -cb '
