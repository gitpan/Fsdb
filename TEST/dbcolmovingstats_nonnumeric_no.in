#h date	epoch count
# test non-numeric fields
19980201        886320000       6
19980202        886406400       8
19980203        886492800       19
19980204        886579200       53
19980205        886665600       20
19980206        886752000       18
19980207        886838400       bogus
19980208        886924800       9
19980209        887011200       22
19980210        887097600       22
19980211        887184000       36
19980212        887270400       bogus
19980213        887356800       bogus
19980214        887443200       6
