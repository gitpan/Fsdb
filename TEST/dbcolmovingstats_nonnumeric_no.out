#fsdb date epoch count moving_mean moving_stddev moving_median
# test non-numeric fields
19980201	886320000	6	6	0	6
19980202	886406400	8	7	1.4142	8
19980203	886492800	19	11	7	8
19980204	886579200	53	21.5	21.764	19
19980205	886665600	20	25	19.442	20
19980206	886752000	18	27.5	17.02	20
19980207	886838400	bogus	27.5	17.02	20
19980208	886924800	9	25	19.27	20
19980209	887011200	22	17.25	5.7373	20
19980210	887097600	22	17.75	6.1305	22
19980211	887184000	36	22.25	11.026	22
19980212	887270400	bogus	22.25	11.026	22
19980213	887356800	bogus	22.25	11.026	22
19980214	887443200	6	21.5	12.261	22
#   | dbcolmovingstats -w 4 -m count
